test/integrations-physical/CrossMarginPhysicalFixture.t.sol

```diff
@@ -9,9 +9,13 @@ import {Pomace} from "pomace/core/Pomace.sol";
 import "pomace/core/PomaceProxy.sol";
 import "pomace/core/PhysicalOptionToken.sol";
 
+import {RolesAuthority} from "entitlements/src/core/RolesAuthority.sol";
+import {RolesAuthorityProxy} from "entitlements/src/core/RolesAuthorityProxy.sol";
+import {Role} from "entitlements/src/config/enums.sol";
+import {MockSanctions} from "entitlements/test/mocks/MockSanctions.sol";
+
 // Mocks
 import "../mocks/MockERC20.sol";
-import {MockWhitelist} from "../mocks/MockWhitelist.sol";
```

Comment:
since `MockWhiteList` is no longer used anywhere, how about we remove it altogether from mocks?

Reply:
i want to keep it around, never know what it may be used for